mavros/src/plugins/waypoint.cpp

```diff
@@ -154,6 +155,10 @@ class WaypointPlugin : public plugin::PluginBase {
 
 		wp_nh.param("pull_after_gcs", do_pull_after_gcs, true);
 
+		wp_nh.param("trig_after_reached", do_signal_reached, false);
+	
+		wp_reached_pub = wp_nh.advertise<std_msgs::UInt16>("reached",1,true);
```

Comment:
Unfortunately std_msgs do not have stamped uint16 msg, so custom WaypointReached.msg is fine.